ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -41,7 +41,7 @@ def login_using_user_and_password(self, user, pwd, project):
         print("Fetch token")
         basic_value = f'{user}:{pwd}'.encode('ASCII')
         basic_value = base64.b64encode(basic_value).decode('ASCII')
-        response = requests.get(self.auth_url, headers={
+        response = requests.get(self.retrieve_auth_url(), headers={
```

Comment:
Where is the `retrieve_auth_url()` function defined?

Reply:
https://github.com/red-hat-data-services/ods-ci/blob/69ef7a06879d81e4016a4dd8fec6467b1b7ed59c/ods_ci/libs/DataSciencePipelinesAPI.py#L204-L210